web-app/src/template.html

```diff
@@ -3,15 +3,11 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width,initial-scale=1.0">
-  <meta name="theme-color" content="#333333">
+  <meta name="theme-color" content="#50AF4F">
```

Comment:
Do you need this in addition to what's in the manifest?

Reply:
According to https://webhint.io/docs/user-guide/hints/hint-meta-theme-color/, yes. I think we'll have to add a number of meta tags for iOS/social sharing as well.

Long-term solution is probably to pull this out of the template and into the `_layout.svelte`, so we can generate it with a config object (JSON file?) that we also pass into the manifest generator plugin.